misc/services/mobile-webui/mobile-webui-frontend/src/containers/activities/distribution/DistributionStepScreen.jsx

```diff
@@ -104,6 +107,20 @@ const DistributionStepScreen = () => {
     );
   };
 
+  const onUnpick = () => {
+    postDistributionReversePicking({
+      wfProcessId,
+      activityId,
+      lineId,
+      stepId,
+    })
+      .then((wfProcess) => {
+        history.goTo(distributionLineScreenLocation);
+        dispatch(updateWFProcess({ wfProcess }));
```

Comment:
not sure, does it matter if the dispatch is performed after or before history.go* ?

Reply:
I want to make sure I'm gone from this screen before the updated job is set to state, as the current step doesn't exist anymore.